install.sh

```diff
@@ -35,7 +35,18 @@ discoverArch() {
 discoverArch
 
 OS=$(echo `uname`|tr '[:upper:]' '[:lower:]')
-curl -# -LO https://github.com/streamnative/pulsarctl/releases/download/$version/pulsarctl-${ARCH}-${OS}
-mv pulsarctl-${ARCH}-${OS} pulsarctl
+TARFILE=pulsarctl-${ARCH}-${OS}.tar.gz
+UNTARFILE=pulsarctl-${ARCH}-${OS}
```

Comment:
The install script is used for downloading different versions of pulsarctl. So the script needs to handle different version's download.